.storybook/storybook.requires.js

```diff
@@ -55,8 +55,6 @@ try {
 const getStories = () => {
   return {
     "./.storybook/stories/Button/Button.stories.js": require("./stories/Button/Button.stories.js"),
-    "./components/buttons/AuthButton/AuthButton.stories.tsx": require("../components/buttons/AuthButton/AuthButton.stories.tsx"),
```

Comment:
Take a look at storybook config. It lloks like storybook lost track of your stories